### PR TITLE
First commit 

### PR DESCRIPTION
**Pull request by tivi2105**
Hi Guru,
Basic setup for our project is done, please review and let me know if any changes are required.
Thanks

### CODE REVIEW

File: src/main/java/com/mrdb/MrdbVersion1Application.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package com.mrdb;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class MrdbVersion1Application {
```
Comment (gurusrikar) [inline src/main/java/com/mrdb/MrdbVersion1Application.java]: Including version numbers in this main application class is a bad idea. When a new version of the app is published, the class name needs to be updated to Version2. So, we should rename this to be more generic
Just - `MrdbApplication`

File: src/mrdb_frontend/jsconfig.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+      ]
+    },
+    "lib": [
+      "esnext",
+      "dom",
+      "dom.iterable",
+      "scripthost"
+    ]
+  }
+}
```
Comment (gurusrikar) [inline src/mrdb_frontend/jsconfig.json:19]: `src/mrdb_frontend/node/node.exe`
This file below should not be committed to git repository. It should only exist in local (add it to gitignore file)

File: src/mrdb_frontend/src/components/HelloWorld.vue [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+      });
+  }
+}
+</script>
+
+<style>
+    h1 {
+      color: blue;
+    }
+</style>
```
Comment (gurusrikar) [inline src/mrdb_frontend/src/components/HelloWorld.vue]: missing a new empty line here

File: src/mrdb_frontend/vue.config.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+        port: 3000,
+        proxy: {
+            '/api': {
+                target: 'http://localhost:8080',
+                ws: true,
+                changeOrigin: true
+            }
+        }
+    }
+}
```
Comment (gurusrikar) [inline src/mrdb_frontend/vue.config.js]: same as above;
missing a new empty line here

File: src/test/java/com/mrdb/MrdbVersion1ApplicationTests.java [hunk 5]
```diff
@@ -0,0 +1,13 @@
+package com.mrdb;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class MrdbVersion1ApplicationTests {
```
Comment (gurusrikar) [inline src/test/java/com/mrdb/MrdbVersion1ApplicationTests.java]: same comment about including version number in class names

**Review by gurusrikar — CHANGES_REQUESTED** [review]
Mostly looks good. Suggested a few changes.

**Comment by tivi2105** [comment]
@gurusrikar suggested changes were made, please take a look and review.

**Review by gurusrikar — APPROVED** [review]

**Pull request merged**